webapp/src/Constants.ts

```diff
@@ -45,6 +45,7 @@ export const Constants = {
         // For a list of Microsoft Graph permissions, see https://learn.microsoft.com/en-us/graph/permissions-reference.
         // Your application registration will need to be granted these permissions in Azure Active Directory.
         msGraphScopes: ['Calendars.Read', 'Mail.Read', 'Mail.Send', 'Tasks.ReadWrite', 'User.Read'],
+        msGraphOboScopes: ['[INCLUDE THE SCOPE FOR THE WEBAPI APP REGISTRATION HERE]'],
```

Comment:
Can we default this to `.default` or put an example in a comment?

Reply:
the default depends on the app registration / tenant so it will need to be changed anyway - I added this in the OBO readme.md file.